src/modules/modCommands/commands/move.js

```diff
@@ -1,4 +1,4 @@
-import { MessageEmbed } from 'discord.js';
```

Comment:
instead of having a whole bunch of copy pasta code from message preview, can't we just create a function "getMessageAsEmbed(message)" and use it in both?

Reply:
We could, but I would say thats out of the scope of this PR.